src/html/scan.js

```diff
@@ -426,11 +426,13 @@ function updateConfig(data, options) {
 
   // set initial visibility status of Serial2 protocol selection
   _('serial1-config').style.display = 'none';
-  data.pwm.forEach((item,index) => {
+if(data.pwm) {
+  data.pwm.forEach((item,index) =>{
     const _pinMode = _(`pwm_${index}_mode`)
     if (_pinMode.value == 14) // Serial2 TX
       _('serial1-config').style.display = 'block';
   });
+}
 
```

Comment:
What happened to the indenting? This can just be `data.pwm?.forEach((item,index) => {` for a smaller changeset, else follow the 4 spaces indenting of the code you're editing (for both the if and the block of cod). Also there should be a space after the if and you took the space out between the `=>{`

Reply:
Thanks Cap'n.
Learnt something today.